ChangeLog

```diff
@@ -1,3 +1,14 @@
+2015-02-25  Hussien Alameldin  <hussien@msu.edu>
+
+   * sandbox/bloom_count.py: renamed to bloom-count.py
+   * sandbox/bloom_count.pyc: renamed to bloom-count.pyc
```

Comment:
the .pyc files are not actually committed in github so you should remove them from the ChangeLog entry.